DSESwift/DataPoint.swift

```diff
@@ -63,48 +63,48 @@ public class DataPoint{
      * Set value of the data point.
      * @param value: value of the datapoint in String.
      **/
-    func setValue(value: String) {
+    public func setValueWithString(value: String) {
         self.value = value
     }
     
     /**
      * Return the timestamp in Int with milliseconds accuracy
      **/
-    func getTimeInMillis() -> Int {
-        return Int(self.time.timeIntervalSince1970 * 1000)
+    public func getTimeInMillis() -> Double {
+        return round(self.time.timeIntervalSince1970 * 1000)
     }
     
     /**
      * Return the value in Int.
      **/
-    func getValueInInt() -> Int {
+    public func getValueInInt() -> Int {
```

Comment:
@kanekin i saw that swift have generics feature, so i wonder if we can introduce 'getValue' function that have functin signature something like

public getValue<T>()